### PR TITLE
Logging changes according to new logging framework 

### PR DESCRIPTION
**Pull request by anmolsachan**
tendrl-bug-id: Tendrl/specifications#28
tendrl-spec: Tendrl/specifications#127
reference-spec: Tendrl/specifications#94
reference code-pr: Tendrl/commons#161

Signed-off-by: anmolsachan asachan@redhat.com

### CODE REVIEW

**Comment by anmolsachan** [comment]
@GowthamShanmugam @shtripat please review

File: MANIFEST.in [hunk 1]
```diff
@@ -2,7 +2,7 @@ include LICENSE
 include AUTHORS
 include ChangeLog
 include requirements.txt
-include tendrl-glusterd.service
+include tendrl-gluster-integration.service
```
Comment (shtripat) [inline MANIFEST.in]: Isnt this change done as part other PR and merged already?

File: tendrl/gluster_integration/manager/__init__.py [hunk 2]
```diff
@@ -23,21 +22,29 @@ def __init__(self):
 
 
 def main():
+    tendrl_ns.publisher_id = "gluster_integration"
```
Comment (shtripat) [inline tendrl/gluster_integration/manager/__init__.py]: Why twice??

File: tendrl/gluster_integration/objects/definition/gluster.py [hunk 3]
```diff
@@ -94,6 +94,15 @@
         integration_id:
           help: "Tendrl managed/generated cluster id for the sds being managed by Tendrl"
           type: String
+        integration_name:
```
Comment (shtripat) [inline tendrl/gluster_integration/objects/definition/gluster.py]: I feel you need to rebase on top of master

Comment (anmolsachan) [inline tendrl/gluster_integration/objects/definition/gluster.py]: I rebased my branch on top of master . I feel Tendrl:spec-28-logging-coverage branch is not upto date with the master .

File: tendrl/gluster_integration/manager/__init__.py (hunk 2 above)
Comment (anmolsachan) [inline tendrl/gluster_integration/manager/__init__.py]: I will fix this

File: MANIFEST.in (hunk 1 above)
Comment (anmolsachan) [inline MANIFEST.in]: I feel Tendrl:spec-28-logging-coverage branch is not upto date with the master . I was working on master . Should I send my changes against Tendrl:spec-28-logging-coverage ?

File: tendrl/gluster_integration/objects/definition/gluster.py (hunk 3 above)
Comment (anmolsachan) [inline tendrl/gluster_integration/objects/definition/gluster.py]: @r0h4n Can u please Tendrl:spec-28-logging-coverage branch? I don't have the permissions I guess

**Pull request closed without merging**